doc/sphinx/source/input.rst

```diff
@@ -298,7 +298,7 @@ A list of the datasets for which a CMORizers is available is provided in the fol
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | ESACCI-FIRE                  | burntArea (Lmon)                                                                                     |   2  | NCL             |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
-| ESACCI-LANDCOVER             | baresoilFrac, cropFrac, grassFrac, shrubFrac, treeFrac (Lmon)                                        |   2  | NCL             |
+| ESACCI-LANDCOVER             | baresoilFrac, cropFrac, grassFrac, shrubFrac, treeFrac (Lmon)                                        |   2  | Python          |
```

Comment:
```suggestion
| ESACCI-LANDCOVER v1.6.1      | baresoilFrac, cropFrac, grassFrac, shrubFrac, treeFrac (Lmon)                                        |   2  | NCL             |
|                              |                                                                                                      |      | (CMORizer       |
|                              |                                                                                                      |      | available until |
|                              |                                                                                                      |      | ESMValTool      |
|                              |                                                                                                      |      | v2.11.0)        |
+------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
| ESACCI-LANDCOVER v2.0.8      | baresoilFrac, cropFrac, grassFrac, shrubFrac, treeFrac (Lmon, frequency=yr)                          |   2  | Python          |
|                              |                                                                                                      |      | (CMORizer       |
|                              |                                                                                                      |      | available since |
|                              |                                                                                                      |      | ESMValTool      |
|                              |                                                                                                      |      | v2.12.0)        |
```